module/zfs/dsl_scan.c

```diff
@@ -1104,7 +1212,7 @@ dsl_scan_zil_block(zilog_t *zilog, blkptr_t *bp, void *arg, uint64_t claim_txg)
 	SET_BOOKMARK(&zb, zh->zh_log.blk_cksum.zc_word[ZIL_ZC_OBJSET],
 	    ZB_ZIL_OBJECT, ZB_ZIL_LEVEL, bp->blk_cksum.zc_word[ZIL_ZC_SEQ]);
 
-	VERIFY(0 == scan_funcs[scn->scn_phys.scn_func](dp, bp, &zb));
+	scan_funcs[scn->scn_phys.scn_func](dp, bp, &zb, B_FALSE);
```

Comment:
Hah.  "did not relocate this block" sounds like some of bprewrite slipping in.  OK.